AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/sample.json

```diff
@@ -16,7 +16,7 @@
 			"conda create --name user_tensorflow -y",
 			"conda activate user_tensorflow",
 			"conda install -c ${ONEAPI_ROOT}/conda_channel tensorflow numpy pyyaml scikit-learn schema lpot ilit matplotlib -y",
-		    "conda install runipy notebook -y"
+		        "conda install runipy notebook flatbuffers -y"
         ],
 		"id": "lpot tensorflow",
 		"steps": [
```

Comment:
Might need to add conda deactivate in the end.

Reply:
I have no guide of sample.json. I don't know where to set the post action in this file.
And I don't find such examples in other sample.json.

From CI/CD view, it will help trouble shooting to keep the running env when case is fault.
The CI system will restore the env when previous case is finished(stoped). And that won't impact next case.
It's handled by CI system.

So every case doesn't need to care to restore the env.